packages/components/inputs/TextInputCustom.tsx

```diff
@@ -82,6 +98,13 @@ export const TextInputCustom = <T extends FieldValues>({
   });
   const inputRef = useRef<TextInput>(null);
 
+  // Passing ref to parent
+  useEffect(() => {
+    if (inputRef.current && setRef) {
+      setRef(inputRef);
+    }
+  }, [setRef]);
```

Comment:
check https://reactjs.org/docs/forwarding-refs.html

Reply:
I'm not sure about the types here : https://github.com/TERITORI/teritori-dapp/blob/dd0f8e38bed58c9b85b10dca862ba29383173240/packages/components/inputs/TextInputCustom.tsx#L61
So i used some ugly `any`.
The ref to forward is defined here : https://github.com/TERITORI/teritori-dapp/blob/dd0f8e38bed58c9b85b10dca862ba29383173240/packages/screens/Swap/components/SwapModalSettings.tsx#L71
Wdyt ?